community/kernel/src/test/java/org/neo4j/test/LogTestUtils.java

```diff
@@ -128,6 +131,63 @@ public static void assertLogContains( FileSystemAbstraction fileSystem, String l
         }
     }
 
+    /**
+     * Opens a {@link LogEntryCursor} over all log files found in the storeDirectory
+     *
+     * @param fs {@link FileSystemAbstraction} to find {@code storeDirectory} in.
+     * @param storeDirectory the store directory where the log files are.
+     * @param logVersionCallback will be kept up to date with which log version the cursor is at.
+     */
+    public static LogEntryCursor openLogs( final FileSystemAbstraction fs, File storeDirectory,
+            final AtomicLong logVersionCallback )
```

Comment:
This current log version callback does not look pretty. Can this method instead return a thing that extends `LogEntryCursor` with a method like `getCurrentLogVersion()`?